plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeExtension.java

```diff
@@ -61,16 +61,15 @@ public JavaRuntimeExtension(ActionManager actionManager,
                                 JavaFqnResolver javaFqnResolver,
                                 JavaRuntimeLocalizationConstant localizationConstant) {
         final DefaultActionGroup mainMenu = (DefaultActionGroup)actionManager.getAction(GROUP_MAIN_MENU);
-        final DefaultActionGroup debugMenu = new DefaultActionGroup(localizationConstant.mainMenuDebugName(), true, actionManager);
```

Comment:
Since we don't have Debug menu now corresponded ```mainMenuDebugName()``` localization constant should be removed.